service/history/replicationDLQHandler.go

```diff
@@ -142,6 +142,13 @@ func (r *replicationDLQHandlerImpl) readMessagesWithAckLevel(
 			ScheduledId:  task.GetScheduledId(),
 		})
 	}
+
+	var replicationTasks []*replicationspb.ReplicationTask
+	pageToken = resp.NextPageToken
```

Comment:
I'm curious why capturing it in a different variable?  why can't you just return resp.NextPageToken like we were doing before?
Also if there is a reason I would rather set pageToken to new value immediately after making the call like on line 130.

Reply:
no specific reason, just easy to read